go/libraries/doltcore/sqle/statsnoms/database.go

```diff
@@ -164,6 +177,63 @@ func (n *NomsStatsDatabase) LoadBranchStats(ctx *sql.Context, branch string) err
 	return nil
 }
 
+func (n *NomsStatsDatabase) ValidateSchemas(ctx *sql.Context, branch string) (bool, error) {
+	root, err := n.sourceDb.GetRoot(ctx)
+	if err != nil {
+		return false, err
+	}
+	tables, err := n.sourceDb.GetTableNames(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	var tags []ref.TagRef
```

Comment:
I don't think this is too dumb but you should use a new ref type for this rather than reusing tags